frontend/src/components/PhoneNumberInput/IntlPhoneNumberInput.tsx

```diff
@@ -64,6 +64,7 @@ export const IntlPhoneNumberInput = forwardRef<
         preventDefaultOnEnter
         {...props}
         placeholder={inputPlaceholder}
+        highContrast={props.highContrast}
```

Comment:
might not be needed since `...props` would already include destructuring of `highContrast`